### PR TITLE
Fixed javadoc warnings

### PR DESCRIPTION
**Pull request by lilgreenbird**

### CODE REVIEW

**Review by v-zhangw — APPROVED** [review]

**Review by peterbae — DISMISSED** [review]

File: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java [hunk 1]
```diff
@@ -288,9 +288,12 @@ private void checkClosed() throws SQLServerException {
                 String sProc = parseProcIdentifier(st.procedureName);
                 try (SQLServerStatement s = (SQLServerStatement) con.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE,
                         ResultSet.CONCUR_READ_ONLY);
-                        SQLServerResultSet rsProcedureMeta = s.executeQueryInternal(
-                                con.isKatmaiOrLater() ? "exec sp_sproc_columns_100 " + sProc + ", @ODBCVer=3, @fUsePattern=0"
-                                                      : "exec sp_sproc_columns " + sProc + ", @ODBCVer=3, @fUsePattern=0")) {
+                        SQLServerResultSet rsProcedureMeta = s
```
Comment (karinazhou) [inline src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java:291]: Can we put `s.executeQueryInternal(con.isKatmaiOrLater()` in one line?

File: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerStatement.java [hunk 2]
```diff
@@ -147,26 +152,35 @@ final void discardLastExecutionResults() {
...
+     * 
+     * @return stmtColumnEncriptionSetting
+     */
     protected SQLServerStatementColumnEncryptionSetting getStmtColumnEncriptionSetting() {
         return stmtColumnEncriptionSetting;
     }
 
     /**
-     * Encapsulates a subset of statement property values as they were set at execution time.
+     * Encapsulates a subset of statement property values as they were set at execution time.F
```
Comment (karinazhou) [inline src/main/java/com/microsoft/sqlserver/jdbc/SQLServerStatement.java:183]: Extra F in the end

File: src/main/java/com/microsoft/sqlserver/jdbc/SharedTimer.java [hunk 3]
```diff
@@ -12,6 +12,31 @@
 import java.util.concurrent.atomic.AtomicLong;
 
 
+/**
+ * Provides timeout handling for basic and bulk TDS commands to use a new shaed timer class. SharedTimer provides a
```
Comment (karinazhou) [inline src/main/java/com/microsoft/sqlserver/jdbc/SharedTimer.java:16]: new shared timer class

File: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java [hunk 4]
```diff
@@ -620,45 +691,52 @@ final int baseYear() {
...
     }
 
+    /** socket factory constructor argument */
     private String socketFactoryConstructorArg = null;
 
     final String getSocketFactoryConstructorArg() {
         return socketFactoryConstructorArg;
     }
 
+    /** trust manage class */
```
Comment (karinazhou) [inline src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java:731]: trust manager class

File: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java (hunk 1 above)
Comment (lilgreenbird) [reply src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java:291]: this is the formatter's doing

**Review by karinazhou — APPROVED** [review]

**Review by peterbae — APPROVED** [review]

**Pull request merged**